rotkehlchen/tasks/calendar.py

```diff
@@ -22,20 +25,29 @@
 from rotkehlchen.db.dbhandler import DBHandler
 from rotkehlchen.db.filtering import EvmEventFilterQuery
 from rotkehlchen.db.history_events import DBHistoryEvents
+from rotkehlchen.errors.asset import UnknownAsset
 from rotkehlchen.errors.misc import InputError
 from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
 from rotkehlchen.logging import RotkehlchenLogsAdapter
 from rotkehlchen.serialization.deserialize import deserialize_hex_color_code
-from rotkehlchen.types import ChainID, OptionalBlockchainAddress, SupportedBlockchain, Timestamp
+from rotkehlchen.types import (
+    ChainID,
+    Location,
+    OptionalBlockchainAddress,
+    SupportedBlockchain,
+    Timestamp,
+    TimestampMS,
+)
 from rotkehlchen.user_messages import MessagesAggregator
-from rotkehlchen.utils.misc import ts_now
+from rotkehlchen.utils.misc import ts_ms_to_sec, ts_now, ts_now_in_ms, ts_sec_to_ms
 from rotkehlchen.utils.mixins.customizable_date import CustomizableDateMixin
 
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 ENS_CALENDAR_COLOR: Final = deserialize_hex_color_code('5298FF')
 CRV_CALENDAR_COLOR: Final = deserialize_hex_color_code('5bf054')
 AIRDROP_CALENDAR_COLOR: Final = deserialize_hex_color_code('ffd966')
+BRIDGE_CALENDAR_COLOR: Final = deserialize_hex_color_code('eeeeee')
```

Comment:
why this color. Can you ask the frontend if they have a pallete for colors to use in calendars?